roles/validations/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
   block:
     - name: Assert all listed validations exist
       ansible.builtin.stat:
-        path: "{{ cifmw_validations_default_path }}{{ item }}"
+        path: "{{ cifmw_validations_default_path }}/{{ item }}"
```

Comment:
you could use `| path_join` ?

Reply:
I think we can just stick with the `/` here yeah?